### PR TITLE
Clarification for https credentials

### PR DESCRIPTION
**Pull request by Jeroendevr**
In an effort to further clarify the documentation. I would like to clarify how to use https in combination with the credentials.

### CODE REVIEW

File: content/docs/deployment/ghpages/contents.lr [hunk 1]
```diff
@@ -34,6 +34,13 @@ rules apply as for the [rsync deployment method :ref](../rsync/).  The HTTPS
 transport on the other hand accepts `--username` and `--password` which
 override the values in the URL.
 
+Example:
+
+```ini
+[servers.ghpages]
+target = ghpages:https://username:pasword@your-user/your-user.github.io
```
Comment (goanpeca) [inline content/docs/deployment/ghpages/contents.lr]: I don't agree with this approach as we are basically advising users how to expose their credentials.

Comment (Jeroendevr) [reply content/docs/deployment/ghpages/contents.lr]: Can you explain how to set up enviroment variables? Cause then that is not clear for me.

Comment (goanpeca) [reply content/docs/deployment/ghpages/contents.lr]: https://www.getlektor.com/docs/deployment/travisci/

**Comment by nixjdm** [comment]
I'm closing this because it's aging and we can't accept it as is. If there is another further clarification that can be done please post another issue or pull request.

Thanks

**Pull request closed without merging**